### PR TITLE
Flip `--incompatible_allow_tags_propagation`

### PR DESCRIPTION
**Pull request by fmeum**
Work towards #8830

### CODE REVIEW

**Comment by fmeum** [comment]
@coeuvre Could you review the flip change, but not merge it yet? The downstream pipeline will run tomorrow morning.

**Review by coeuvre — COMMENTED** [review]
LGTM. I will mark this as approved after the downstream pipeline is finished.

**Comment by fmeum** [comment]
@coeuvre Downstream looks good: https://buildkite.com/bazel/bazelisk-plus-incompatible-flags/builds/1540

**Review by coeuvre — APPROVED** [review]

**Pull request closed without merging**
